Extensions/dnSpy.AsmEditor/MethodBody/CilBodyVM.cs

```diff
@@ -316,6 +317,16 @@ public LocalsIndexObservableCollection(CilBodyVM owner, Func<LocalVM> createNewI
 
 		bool ReplaceInstructionWithNopCanExecute(InstructionVM[] instrs) => instrs.Any(a => a.Code != Code.Nop);
 
+		void ReplaceInstructionWithMultipleNops(InstructionVM[] instrs) {
```

Comment:
This method should be similar to `ReplaceInstructionWithNop()`, it has some optimizations to prevent every other instuction from updating while updating/adding instructions. This is important if it's a huge method body, not if it's a normal sized method body.